src/Symfony/Component/Form/FormEvents.php

```diff
@@ -12,7 +12,7 @@
 
 /**
  * To learn more about how form events work check the documentation
- * entry at {@link http://symfony.com/doc/any/components/form/form_events.html}
+ * entry at {@link http://symfony.com/doc/any/components/form/form_events.html}.
```

Comment:
I didn't say that the docs are correct. But this is not the part of this PR.

BTW, I like
> line long by some parsers.

part ;)

My point is that this PR is not mentioned to fulfill docs and add missing short docs where there are none.
This PR fix some issues raised by PHP CS Fixer to let the branch be consistent with it to let fabbot.io do not warn about it any more.